src/Twig/Extension/RequirementExtension.php

```diff
@@ -62,14 +68,17 @@ public function formatVersions($data): array
         if (reset($elements) instanceof Requirement) {
             $this->normalizeVersionHelper($elements);
         } else {
-            foreach ($elements as &$category) {
-                $this->normalizeVersionHelper($category);
+            foreach ($elements as &$element) {
```

Comment:
Auto-rename gone awry.  $category is better if that's what it actually is.  If anything, $elements is the problematic name here.

Reply:
Yep, also thought about renaming $elements to $categories.